#axis:
cmd_x: 1
cmd_y: 3
cmd_z: 0
cmd_joint: 1
cmd_gripper_rot: 7
cmd_gripper_grip: 6

#buttons
cmd_a_positive: 5
cmd_a_negative: 4
crawler: 1
enable: 4
fast: 5
joint_increase: 2
joint_decrease: 0
control_mode_toggle: 3
